[2023-11-23T03:41:53.910+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.fact_to_cloud scheduled__2023-11-04T00:00:00+00:00 [queued]>
[2023-11-23T03:41:53.967+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.fact_to_cloud scheduled__2023-11-04T00:00:00+00:00 [queued]>
[2023-11-23T03:41:53.968+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-23T03:41:54.034+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fact_to_cloud> on 2023-11-04 00:00:00+00:00
[2023-11-23T03:41:54.079+0000] {standard_task_runner.py:57} INFO - Started process 490 to run task
[2023-11-23T03:41:54.106+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'fact_to_cloud', 'scheduled__2023-11-04T00:00:00+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmpo3he7a89']
[2023-11-23T03:41:54.133+0000] {standard_task_runner.py:85} INFO - Job 136: Subtask fact_to_cloud
[2023-11-23T03:41:54.336+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.fact_to_cloud scheduled__2023-11-04T00:00:00+00:00 [running]> on host e647c96745b0
[2023-11-23T03:41:54.703+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='fact_to_cloud' AIRFLOW_CTX_EXECUTION_DATE='2023-11-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-04T00:00:00+00:00'
[2023-11-23T03:41:56.367+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-23T03:41:56.534+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_dag, task_id=fact_to_cloud, execution_date=20231104T000000, start_date=20231123T034153, end_date=20231123T034156
[2023-11-23T03:41:56.715+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-23T03:41:56.915+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
